homeassistant/components/ddwrt/device_tracker.py

```diff
@@ -46,6 +56,8 @@ def __init__(self, config):
         self.host = config[CONF_HOST]
         self.username = config[CONF_USERNAME]
         self.password = config[CONF_PASSWORD]
+        self.type = config[CONF_TYPE]
+        self.monitored_conditions = config[CONF_MONITORED_CONDITIONS]
```

Comment:
I don't think this is needed.

Please see this architectural decision:

https://github.com/home-assistant/architecture/blob/master/adr/0003-monitor-condition-and-data-selectors.md

All data is fetched on line # 204 in a single call.

Reply:
DDWRT doesn't seem to offer any other way to get the data requested on line #204, which is the list of all wireless clients.
The type parameter allow the user to select which clients they would like to monitor.

Reply:
I have no idea why I didn't get a notification for your comments, I apologize.